tdrs-frontend/src/components/FormGroup/FormGroup.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+function FormGroup({
```

Comment:
Should this get some class-level documentation, especially since it's designed as a reusable component?

Reply:
Yes, it should. I'll add that in.

Reply:
@alexsoble I added those docs. Let me know if something isn't clear or if anything needs more information. Thanks!

Reply:
Thanks, looks good!